### DISCUSSION TITLE
可以下载不能转发频道的文件吗

### DISCUSSION THREAD

**Discussion by rebornll** [Dineshkarthik/telegram_media_downloader#339] [Q&A]
Can I download files that cannot forward channels

**Comment by tangyoha** [comment]
@rebornll hi, i think it will be not work. because telegram api not allow to download.

**Reply by androiddisk** [reply]
好伤心 不能下载不能转发的文件

**Reply by tangyoha** [reply]
see https://github.com/tangyoha/telegram_media_downloader ,you can download files that cannot forward channels.这个可以下载，前提是你得在那个群里面
